### ISSUE TITLE
A11y_SQLAzureDataStudio_Extension_Install-UninstallExtension_Keyboard: Tool tip assigned to the texts is not visible on keyboard focus.

### ISSUE THREAD

**Issue by v-atkulk** [microsoft/azuredatastudio#9387]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

#A11yMAS; #A11yTCS; #SQL Azure Data Studio; #DesktopApp; #Win32; #MAS1.1.1; #Benchmark; #A11y_SQLAzureDataStudio; #A11ySev2; #Keyboard;

**Environment Details:**
Version: 1.16.0-insider
Commit: 506c6a5
Date: 2020-02-17T05:43:18.399Z
VS Code: 1.42.0
Electron: 7.1.11
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363

**Additional Details:**
MAS reference: MAS1.1.1

**Repro Steps:**
1.Launch Azure Data Studio.
2.From the left menu, click the 'Extensions' icon to open the Extension Manager.
3.Choose any extension from the library.
4.Click the green 'Install' button to install.
5.Launch screen reader and navigate to the content present on the top of the screen.
6.Check if tool tip is visible after focusing the control/text.

**Actual:**
Tool tip is not visible when the controls/elements receive keyboard focus.

**Expected:**
Tool tip assigned to all the contents/controls should be visible when the controls receive keyboard focus.

**User Impact:**
Keyboard as well as screen reader users will not receive the tool tip information that is assigned to the texts/controls present on the screen.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets: https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**MAS Reference:
MAS 1.1.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={d2d2051f-bdc8-4af7-8e18-38aae867e216}

**Attachment link reference:**
https://microsoftapc-my.sharepoint.com/:v:/g/personal/v-atkulk_microsoft_com/EXglgQGg86JHj3DacRU5AMUBqvltBc2-qL_0opoltCpEWg?e=dxd3Tm

**Comment by alanrenmsft** [comment]
this is a vscode issue, please open an issue in vscode repo.

**Comment by hannahqin** [comment]
@v-atkulk This is also a duplicate of an upstream bug that was previously approved for exclusion: http://sqlbuvsts01:8080/Main/SQL%20Server/_workitems#_a=edit&id=11697790. It is already documented in our Conformance Document.

**Comment by Harika1702** [comment]
Lowering the Severity to Sev3 as per the discussion with our accessibility SME. As they tooltips are not that much important, and button name itself conveying the information to the users.
GitHub Tags:#Sev3;

**Comment by Vishnuvarthancr7** [comment]
GitHubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#SQLADS-Win32-Feb2020;
